utils/simulation/sc_linac_physics_service.py

```diff
@@ -103,16 +121,19 @@ def __init__(self):
                     AutoSetupCMPVGroup(prefix=cm_prefix + "00:", cm_name=cm_name)
                 )
 
+                rfs_prefix = cm_prefix + "00:"
+
                 for cav_num in range(1, 9):
                     cav_prefix = cm_prefix + f"{cav_num}0:"
 
-                    jt_prefix = f"CLIC:CM{cm_name}:3001:PVJT:"
-                    liquid_level_prefix = f"CLL:CM{cm_name}:"
-
                     HOM_prefix = f"CTE:CM{cm_name}:1{cav_num}"
 
-                    cavityGroup = CavityPVGroup(prefix=cav_prefix, isHL=is_hl)
+                    cavityGroup = CavityPVGroup(
+                        prefix=cav_prefix, isHL=is_hl, heater_group=heater_group
+                    )
                     self.add_pvs(cavityGroup)
+                    cm_group.add_cavity(cavityGroup)
```

Comment:
If you end up changing it to a dictionary, this line can directly be `cm_group.cavities[cav_num] = cavityGroup`